cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -429,3 +429,16 @@ func DeepHashObject(hasher hash.Hash, objectToWrite interface{}) {
 	hasher.Reset()
 	fmt.Fprintf(hasher, "%v", dump.ForHash(objectToWrite))
 }
+
+// IsPodReady checks if a pod is ready.
+// It iterates through the conditions of the pod's status and returns true if it finds a condition of type "PodReady" with status "ConditionTrue".
+// Otherwise, it returns false.
+func IsPodReady(pod *v1.Pod) bool {
```

Comment:
i don't think we need the new helper function. just use the code inside etcd.go

Reply:
@neolit123 ok. got it.

Reply:
@neolit123 done. thanks so much.